solutions.html

```diff
@@ -0,0 +1,147 @@
+          <div class="carousel__list solutions applications">
+            {% for service in site.data.solutions.industries.applications %}
+              <label class="carousel__item" for="solution-{{ service.index }}-2" id="card-{{ service.index }}-solution-2">
+                <div class="carousel__title services__title inria">{{ service.title }}</div>
+                <div class="carousel__description">{{ service.description }}</div>
+                <div class="carousel__button">
+                  <a class="button" href="/">{{site.data.solutions.industries.more_button}}</a>
+                </div>
+              </label>
+            {% endfor %}
+          </div>
+        </div>
+        <p class="title-3">{{site.data.solutions.industries.subtitle_2}}</p>
+        <h2 class="title-2">{{site.data.solutions.project_development.title}}</h2>
+        <p class="title-3">{{site.data.solutions.project_development.description}}</p>
+      </div>
+    </div>
+    <div class="solutions__project-development">
+      <div class="wrapper timeline">
+        {% for item in site.data.solutions.project_development.approaches %}
+        <div class="timeline-item">
+          <div class="timeline-item__icon">
+            <img src="assets/images/solutions/{{ item.image }}.webp" alt="Approach">
+          </div>
+          <div class="timeline-item__content">
+            <div class="timeline-item__content-title inria">{{ item.title}}</div>
+            <div class="timeline-item__content-description">{{ item.description}}</div>
+          </div>
+          <div class="timeline__processes">
+            {% for process in item.processes%}
+              <div class="timeline__processes-item">
+                {{process.item}}
+                <div class="arrow">→</div>
+              </div>
+            {% endfor %}
+          </div>
+        </div>
+      {% endfor %}
+      </div>
+    </div>
+    <div class="solutions__staff-augmentation">
+      <div class="row wrapper">
+        <h3 class="title-2">{{ site.data.solutions.staff_augmentation.title }}</h3>
+        <p class="base-content">{{ site.data.solutions.staff_augmentation.description }}</p>
+        <div class="timeline wrapper">
+          {% for item in site.data.solutions.staff_augmentation.approaches %}
+            <div class="timeline-item">
+              <div class="timeline-item__icon">
+                <img src="assets/images/solutions/{{ item.image }}.webp" alt="Approach">
+              </div>
+              <div class="timeline-item__content">
+                <div class="timeline-item__content-title inria">{{ item.title}}</div>
+                <div class="timeline-item__content-description">{{ item.description}}</div>
+              </div>
+              <div class="timeline__processes">
+                {% for process in item.processes%}
+                  <div class="timeline__processes-item">
+                    {{process.item}}
```

Comment:
please check the images:

![image](https://github.com/katatsu12/live_blog/assets/22080708/f2b9f5dd-320b-446f-91ba-f2367622128e)

![image](https://github.com/katatsu12/live_blog/assets/22080708/d03440cb-a214-4381-9534-2314afd3cc1e)

Reply:
Will check